lib/pry/commands/hist.rb

```diff
@@ -170,8 +170,9 @@ def find_history
           else
             Pry.history.to_a.last(Pry.history.session_line_count)
           end
-      # The last value in history will be the 'hist' command itself.
-      Pry::Code(h[0..-2])
+
+      # The last value in history will be the 'hist' command itself
```

Comment:
I removed it since the last value could be anything now... no need to make an annotation about it, let me know if this is ok with you.